packages/__docs__/src/Playground/index.tsx

```diff
@@ -215,8 +211,16 @@ class Playground extends Component {
 
     return (
       <AppContext.Consumer>
-        {({ library, themeKey, themes }) => (
-          <div css={styles.playground}>
+        {({
+          library,
+          themeKey,
+          themes
+        }: {
+          library: Record<string, any>
+          themeKey: string
+          themes: Record<string, any>
+        }) => (
```

Comment:
These are coming from the context (AppContext), but that part of the code is not typed yet. Please add a TODO:

`// TODO: update type once AppContext is typed`